content/en/docs/commands.md

```diff
@@ -5,15 +5,15 @@ description: >
     Usage of each rebar3 command.
 ---
 
-Each command represents a task which runs one or more providers to fulfill the task.
+Each command represents a task which runs one or more providers to fulfil the task.
```

Comment:
```suggestion
Each command represents a task which runs one or more providers to fulfill the task.
```
I'd probably keep the US variant here.

Reply:
Oh, I actually thought that WAS the US variant :) sorry about that. TIL

Reply:
I knew I wanted to keep US for the use of "z" instead of "s", everywhere... 😄